server.go

```diff
@@ -11,7 +11,10 @@ import (
 	"net/http"
 )
 
-var db bolt.DB
+var (
+	db bolt.DB
```

Comment:
@hvnsweeting  should be `db *bolt.DB`? https://github.com/boltdb/bolt/blob/master/db.go#L150